packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -123,13 +165,13 @@ export interface ComboBoxProps
   /**
    * Additional props passed to Downshift
    */
-  downshiftProps?: React.ComponentProps<typeof Downshift>;
+  downshiftProps?: ComponentProps<typeof Downshift<ItemType>>;
 
   /**
    * Provide helper text that is used alongside the control label for
    * additional help
    */
-  helperText?: string;
+  helperText?: ReactNode;
```

Comment:
Shouldn't we change this to `string` as it was before?

Reply:
I was following the way the things are in `Dropdown` component and there the `helperText` is a `ReactNode`. I don't mind making it a string as it was but sometimes its usefull to have it as a node. WDYT?

Reply:
It's not just Dropdown, every other component with a `helperText` prop has it as a node. I think it's a good idea to move it to be a node.